lib/requestContextPlugin.js

```diff
@@ -17,11 +17,16 @@ function plugin(fastify, opts, next) {
   const hook = opts.hook || 'onRequest'
 
   fastify.addHook(hook, (req, res, done) => {
+    const defaultStoreValues =
```

Comment:
Maybe let's move it inside the runWith block to eliminate one closure

Reply:
This values are second argument of the `runWith`.